docs/contributors_guide/maintenance/simp_puppet_module_release_procedures/Release_GitHub_and_Deploy_PuppetForge.rst

```diff
@@ -24,7 +24,7 @@ To create the releases from an annotated tag:
       bundle update
       bundle exec rake changelog_annotation > foo
 
-#. Create the annotated tag.  In this example the content of 'foo' is::
+#. Create the annotated tag.  In this example the content of 'foo' is:
```

Comment:
@trevor-vaughan  :: is a formatting construct to have the remaining content in a text block.  Have you verified this renders appropriately?

Reply:
Fixed